src/commander/commander_btc.c

```diff
@@ -28,70 +28,86 @@ static const char* _coin_tbtc = "BTC Testnet";
 static const char* _coin_ltc = "Litecoin";
 static const char* _coin_tltc = "LTC Testnet";
 
-commander_error_t commander_btc_pub(const BTCPubRequest* request, PubResponse* response)
+// Returns the string to be used in the confirm title. Returns NULL for an invalid coin.
+static const char* _coin_title(BTCCoin coin)
 {
-    if (!app_btc_enabled(request->coin)) {
```

Comment:
Why is this removed?

Reply:
The diff is just confusing, the line is still there in commander_btc_pub()